velox/exec/tests/utils/AggregationFuzzerUtils.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <boost/random/uniform_int_distribution.hpp>
+#include "velox/exec/tests/utils/InputGenerator.h"
+#include "velox/exec/tests/utils/ResultVerifier.h"
+
+namespace facebook::velox::exec::test {
+
+struct AggregationFuzzerOptions {
```

Comment:
Once filterSignatures are moved into FuzzerToolkit, this will be the only thing left here. We can then rename the file to AggregationFuzzerOptions.h